python/ray/tests/test_state_api_log.py

```diff
@@ -595,26 +596,33 @@ def verify():
             # When glob filter is not provided, it should provide all logs
             logs = list_logs(node_id=node["node_id"])
             assert "raylet" in logs
-            assert "gcs_server" in logs
             assert "dashboard" in logs
             assert "agent" in logs
             assert "internal" in logs
             assert "driver" in logs
             assert "autoscaler" in logs
 
+            # TODO(rickyyx): When we have multiple nodes on the same ip,
```

Comment:
This is why the accidentally merged PR didn't fail the test. I think this is actually a bug, i.e, how to handle nodes with same node-ip but different node-ids.

Reply:
Can you create an issue?